src/components/ConnectToNerbordButton/ConnectToNerdbordButton.tsx

```diff
@@ -0,0 +1,41 @@
+﻿'use client'
+import { connectToNerdbord } from '@/actions/user/connectToNerdobord'
+import { Button } from '../Button/Button'
+import RotateIcon from '@/assets/icons/RotateIcon'
+import { ToastStatus, useToast } from '@/contexts/ToastContext'
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+
+export const ConnectToNerdbordButton = () => {
+  const { addToast } = useToast()
+  const [loading, setLoading] = useState(false)
+  const router = useRouter()
+
+  const handleConnectToNerdbord = async () => {
+    setLoading(true)
+    try {
+      await connectToNerdbord()
+      router.refresh()
+    } catch (error) {
+      console.error(error)
+      addToast(
+        'User not found. Make sure your Github account is used on Nerdbord',
```

Comment:
**nitpick:** You should be able to get this message from error, `addToast(error.message)`

Reply:
In server action, I have just thrown the error as a string and I don't have a message prop there. I can do something like `addToast(String(error))` but I get a message like this: `Error: User not found. Make sure your GitHub account is used on Nerdbord`.